src/pixelator/cli/analysis.py

```diff
@@ -60,23 +60,24 @@
     "--polarization-normalization",
     default="clr",
     required=False,
-    type=click.Choice(["raw", "clr", "denoise"]),
+    type=click.Choice(["raw", "clr"]),
     show_default=True,
     help=(
         "Which approach to use to normalize the antibody counts:"
         " \n\traw will use the raw counts\n\tclr will use the"
-        " CLR transformed counts\n\tdenoise will use"
-        " CLR transformed counts and subtract the counts of the"
-        " control antibodies"
+        " CLR transformed counts"
     ),
 )
 @click.option(
-    "--polarization-binarization",
+    "--polarization-permutations",
+    default=None,
     required=False,
-    is_flag=True,
+    type=click.IntRange(),
```

Comment:
You probably need to define a `min` here to make it work. See the other colocalization definitions.

Reply:
Ok, thanks!